README.md

```diff
@@ -20,3 +20,48 @@ Please refer to the [section](CONTRIBUTING.md#adding-a-new-project) in the contr
 ## Contributing to this Repository
 
 For information on how to contribute to this project, please read [CONTRIBUTING.md](CONTRIBUTING.md) for contribution guidelines.
+
+## Build Instructions
+
+1. Clone repository (first time) with `git clone` or pull updates to the repository (subsequent times) with `git pull`
+2. Install dependencies: `rush install`
+3. Clean: `rush clean`
+4. Rebuild source: `rush rebuild`
+5. Run tests: `rush test`
+
+The `-v` option for `rush` is short for `--verbose` which results in a more verbose command.
+
+The above commands iterate and perform their action against each package in the monorepo.
+
+For incremental builds, the `rush build` command can be used to only build packages that have changes versus `rush rebuild` which always rebuilds all packages.
+
+> Note: It is a good idea to `rush install` after each `git pull` as dependencies may have changed.
+
+## Source Code Edit Workflow
+
+1. Make source code changes on a new Git branch
+2. Ensure unit tests pass when run locally: `rush test`
+3. Locally commit changes: `git commit` (or use the Visual Studio Code user interface)
+4. Repeat steps 1-3 until ready to push changes
+5. Add changelog entry (which could potentially cover several commits): `rush change`
+6. Follow prompts to enter a change description or press ENTER if the change does not warrant a changelog entry. If multiple packages have changed, multiple sets of prompts will be presented.
+7. Completing the `rush change` prompts will cause new changelog entry JSON files to be created.
+8. Commit the changelog JSON files.
+9. Publish changes on the branch and open a pull request.
+
+> Note: The CI build will break if changes are pushed without running `rush change`. The fix will be to complete steps 5 through 9.
+
+Here is a sample [changelog](https://github.com/Microsoft/web-build-tools/blob/master/apps/rush/CHANGELOG.md) to demonstrate the level of detail expected.
```

Comment:
Can you update this link?  It's currently broken in the iModel.js repo too.

Reply:
Updated link is here, https://github.com/microsoft/rushstack/blob/master/apps/rush/CHANGELOG.md

Reply:
nice catch